client/my-sites/media-library/list-item.jsx

```diff
@@ -20,12 +19,15 @@ import ListItemAudio from './list-item-audio';
 import ListItemDocument from './list-item-document';
 import { getMimePrefix } from 'lib/media/utils';
 import EditorMediaModalGalleryHelp from 'post-editor/media-modal/gallery-help';
+import canCurrentUser from 'state/selectors/can-current-user';
+import { getSelectedSiteId, getSectionName } from 'state/ui/selectors';
+
 /**
  * Style dependencies
  */
 import './list-item.scss';
 
-export default class extends React.Component {
+class MediaLibraryListItem extends React.Component {
 	static displayName = 'MediaLibraryListItem';
 
 	static propTypes = {
```

Comment:
New props needs to be appended to `propTypes` to avoid unknown attribute DOM errors via `...props`:
```
isMediaLibrary: PropTypes.bool,
isContributor: PropTypes.bool,
```